lambdas/gro-pull-file/src/main/java/uk/gov/di/data/lep/GroPullFile.java

```diff
@@ -79,27 +87,38 @@ private void transferFile(String xmlBucket) {
             client.connect(host);
             client.authPublickey(username, privateKeyProvider);
 
-            logger.info("Pulling file {} from GRO", groFileName);
```

Comment:
I think we still want this logging in place - it fulfils a bit of a different purpose to the audit queue